javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -71,6 +71,15 @@ public void setup() {
                    && Collections.equals(clojurePersistent, javaMutable)
                    && Collections.equals(asJavaCollection(scalaPersistent), javaMutable)
                    && Collections.equals(slangPersistent, javaMutable);
+
+            Memory.storeMemoryUsages(ELEMENTS, Base.this,
```

Comment:
Should I apply this to the other collections also?
@danieldietrich, @ruslansennov, @eduardmanas?
(please see the results for `Vector` above, in the commit message)